polling_stations/apps/file_uploads/views.py

```diff
@@ -173,6 +173,7 @@ def post(self, request, *args, **kwargs):
                 Upload.objects.get_or_create(
                     gss=council, election_date=election_date, timestamp=now
                 )
+
```

Comment:
Can you remove this - saves future confusion if using  `git blame -- polling_stations/apps/file_uploads/views.py`.